chain_info/Mangata_X.yaml

```diff
@@ -0,0 +1,42 @@
+name: Mangata X
+network_type: mainnet
+chain:
+  type: parachain
+  consensus: PoS
+  sudo: true
+  para_id: 2110
+chainspec:
+  http_url: https://raw.githubusercontent.com/mangata-finance/chainspec/master/mangata-kusama-mainnet.json
+goals:
+  - An interchain DEX with the most capital-efficient and fair price structure for traders, liquidity providers and arbitrageurs.
+repository: https://github.com/mangata-finance/mangata-node
+validators:
+  - Community nodes
+release_cycle: roughly once a month
+specs:
+  block_time: 12
+  ed: 10000000000
+  decimals: 18
+  token: MGX
+  ss58_format: 42
+contacts:
+  - type: email
+    contact: "hello@mangata.finance"
+rpc_endpoints:
+  - name: Parity
+    url: wss://kusama-rpc.mangata.online:443
```

Comment:
This is definitely NOT a Parity endpoint.

Reply:
I changed line 26

https://github.com/paritytech/polkadot_network_directory/pull/60/commits/4d671b99cb19ccf0a171393e7981f322a9d04d9e